integration_tests/__tests__/generate_coverage_for_files.test.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+const path = require('path');
+const os = require('os');
+const skipOnWindows = require('../../scripts/skip_on_windows');
+const {cleanup, writeFiles, extractSummary} = require('../utils');
+const runJest = require('../runJest');
+
+const DIR = path.resolve(os.tmpdir(), 'generate_coverage_for_files_test');
+
+skipOnWindows.suite();
+
+beforeEach(() => cleanup(DIR));
+afterAll(() => cleanup(DIR));
+
+test('--generateCoverageForFiles', () => {
+  writeFiles(DIR, {
+    '.watchmanconfig': '',
```

Comment:
i think if you have watchman installed on your computer (which we have at fb) it'll complain in stdout if this file is not present, which will in turn make it harder to snapshot the output :(